### PR TITLE
Fixed special cancelled po_task case

### PR DESCRIPTION
**Pull request by mkerns355**
Partially Resolves #[1946](https://vajira.max.gov/browse/CASEFLOW-1946) and #[2570](https://vajira.max.gov/browse/CASEFLOW-2570)

### Description
Cleaned up dead code and some comments from this previous PR (#16951). Added changes to CAMO and Program Office Tab Names. Additional columns will be added to this PR (# )

### Acceptance Criteria
- [ ] Code compiles correctly

### CODE REVIEW

File: app/models/organizations/vha_camo.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def queue_tabs
   end
 
   def in_progress_tasks_tab
```
Comment (bgantick) [inline app/models/organizations/vha_camo.rb:21]: discrepancy between names here

File: app/models/queue_tab.rb [hunk 2]
```diff
@@ -179,8 +167,8 @@ def assignee_is_organization
 require_dependency "organization_on_hold_tasks_tab"
 require_dependency "organization_tracking_tasks_tab"
 require_dependency "organization_unassigned_tasks_tab"
-require_dependency "vha_program_office_assigned_tasks_tab"
-require_dependency "vha_program_office_in_progress_tasks_tab"
-require_dependency "vha_program_office_ready_for_review_tasks_tab"
-require_dependency "vha_program_office_on_hold_tasks_tab"
-require_dependency "vha_program_office_completed_tasks_tab"
+# require_dependency "vha_program_office_assigned_tasks_tab"
```
Comment (bgantick) [inline app/models/queue_tab.rb:170]: Remove commented dependencies?

File: app/models/queue_tabs/vha_program_office_completed_tasks_tab.rb [hunk 3]
```diff
@@ -4,23 +4,35 @@ class VhaProgramOfficeCompletedTasksTab < QueueTab
...
   def tasks
-    Task.includes(*task_includes).visible_in_queue_table_view
-      .closed
-      .where(assigned_to: assignee)
+    Task.includes(*task_includes).visible_in_queue_table_view.where(
+      id: closed_tasks.map(&:id) - parent_ids_with_cancelled_assess_documentation_task
+    )
   end
 
+  # def tasks
```
Comment (bgantick) [inline app/models/queue_tabs/vha_program_office_completed_tasks_tab.rb:30]: remove?

File: client/COPY.json [hunk 4]
```diff
@@ -269,13 +269,16 @@
   "USER_QUEUE_PAGE_ASSIGNED_TASKS_DESCRIPTION": "Cases assigned to you:",
   "USER_QUEUE_PAGE_ON_HOLD_TASKS_DESCRIPTION": "Cases on hold (will return to \"Assigned\" tab when hold is completed):",
   "QUEUE_PAGE_COMPLETE_TASKS_DESCRIPTION": "Cases completed (last 7 days):",
+  "VHA_QUEUE_PAGE_COMPLETE_TASKS_DESCRIPTION": "Cases completed:",
 
-  "ORGANIZATIONAL_QUEUE_PAGE_IN_PROGRESS_TAB_TITLE": "In Progress Tasks",
```
Comment (bgantick) [inline client/COPY.json:273]: are these not in use in other org queues?

**Review by bgantick — CHANGES_REQUESTED** [review]

File: app/models/queue_tab.rb (hunk 2 above)
Comment (mkerns355) [reply app/models/queue_tab.rb:170]: Will be removed as determined to not be needed.

File: client/COPY.json (hunk 4 above)
Comment (mkerns355) [reply client/COPY.json:273]: Yep not being used any else in the code base.

**Review by bgantick — APPROVED** [review]

**Pull request merged**